AdobeStockAssetApi/Api/Data/AssetInterface.php

```diff
@@ -23,10 +25,10 @@ interface AssetInterface extends \Magento\Framework\Api\ExtensibleDataInterface
     const ADOBE_ID = "adobe_id";
     const CREATED_AT = 'created_at';
     const UPDATED_AT = 'updated_at';
-    const MEDIA_TYPE_ID = 'media_type_id';
+    const MEDIA_TYPE = 'media_type';
```

Comment:
Are you sure it is necessary to introduce `media_type` and `premium_level` extensible entities?

It's enough to have hardcoded id to name for these fields in my opinion

Reply:
@sivaschenko  I decided to introduce this implementation because of the next: the asset looks like a complex object consists of the several required components: the category, creator, media type and premium level.
They have different complicity structure but in fact, they all are required to be saved and provide internal id's to link them into the asset object.

Having this as a point I decided to use the strategy for saving images (preview and licensed in future) where we, at the first step, save asset components and then asset with the related entities. From my point of view, having this approach we provide more flexibility for managing the asset save process, use components independently for different scenarios (I think we will need this when the licensed image part will be in progress), cover with tests should be less complicated.